kernel/section.mli

```diff
@@ -68,27 +68,16 @@ val all_poly_univs : 'a t -> Univ.Level.t array
    constraints about monomorphic universes, which prevent declaring
    monomorphic globals. *)
 
-type abstr_info = private {
```

Comment:
I've realized that we lost the private flag by setting this in Declarations. This was ensuring that the only way to build this data was in Section, but I don't know how much it matters.

Reply:
Considering it either was immediately destructed in safe_typing or went outside the kernel I don't think it brought anything.

Reply:
Fair enough.